Feat(web, web-react, web-twig): Vypnout/zapnout zavírání Modalu při kliku na backdrop #DS-731 (#1077)
lmc-eu/spirit-design-system: packages/web-twig/src/Resources/components/Modal/README.md

### API

| Name                   | Type     | Default | Required | Description                                                  |
| ---------------------- | -------- | ------- | -------- | ------------------------------------------------------------ |

> | `closeOnBackdropClick` | `bool`   | `true`  | ✕        | Allows/disallows modal closure when clicking on the backdrop |

Reviewer: ```suggestion
| `closeOnBackdropClick` | `bool`   | `true`  | ✕        | Whether the modal will close when backdrop is clicked |
```

Only suggestion, we often use `Whether` word for boolean props to describe the possibility to something switch on/off or allow/disallow.

Author: changed to `Whether …` sentence
